OMCompiler/Compiler/Template/CodegenOMSIC.tpl

```diff
@@ -143,7 +143,9 @@ template createMakefile(SimCode simCode, String target, String makeflieName)
     let includedir = '<%fileNamePrefix%>.fmutmp/sources/include/'
     let mkdir = match makefileParams.platform case "win32" case "win64" then '"mkdir.exe"' else 'mkdir'
     let OMLibs = match makefileParams.platform case "win32" case "win64" then 'lib' case "linux64" then 'lib/x86_64-linux-gnu' else 'lib'
-    let lapackDirWin = match makefileParams.platform case "win32" then '$(OMDEV)/tools/msys/mingw32/bin' case "win64" then '$(OMDEV)/tools/msys/mingw64/bin' else ''
+    // TODO AHeu: Is OMDEV even set on installed OpenModelica???
```

Comment:
The installer checks if you have the OMDEV variable set. If so, it asks to removed it and if you say no, it tells you to keep OMDEV updated.